documentation/asciidoc/computers/config_txt/codeclicence.adoc

```diff
@@ -2,7 +2,9 @@
 
 Hardware decoding of additional codecs on the Raspberry Pi 3 and earlier models can be enabled by https://codecs.raspberrypi.com/license-keys/[purchasing a licence] that is locked to the CPU serial number of your Raspberry Pi.
 
-On the Raspberry Pi 4, the hardware codecs for MPEG2 or VC1 are permanently disabled and cannot be enabled even with a licence key; on the Raspberry Pi 4, thanks to its increased processing power compared to earlier models, MPEG2 and VC1 can be decoded in software via applications such as VLC. Therefore, a hardware codec licence key is not needed if you're using a Raspberry Pi 4.
+The Raspberry Pi 4 has permanently disabled hardware decoders for MPEG2 and VC1. These codecs cannot be enabled, so a hardware codec licence key is not needed. Software decoding of MPEG2 and VC1 files performs well enough for typical use cases.
```

Comment:
I think it's better to not mention that these hardware blocks exist on the Pi 4 series; they also don't exist on Pi 5 series devices. Perhaps something like `Raspberry Pi 4 and 5 do not use hardware decode for MPEG2 and VC1; software decoding performs well enough for typical use cases.`

Reply:
This paragraph talks about the Pi 4, so I don't want to make things messy by talking about the 5 in the last sentence. IMO the cat's already out of the bag that these disabled decoders exist, since it's already public information in the docs. So we might as well acknowledge it so anybody who knows about those disabled decoders can find out that they're impossible to use, even with a licence.

@aallan does that seem reasonable to you? I can remove mention of the Pi 4 disabled decoders if you feel that it isn't useful. I just don't want to remove useful information that's already in the docs.

Reply:
Keep it.